frontend/src/views/camp/Invitation.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <auth-container>
+    <div v-if="campCollaborationFound && campCollaborationOpen !== false">
+      <h1 class="display-1">{{ this.$tc('components.invitation.title') }} {{ camp ? camp.title : '' }}</h1>
+
+      <v-spacer />
+      <div v-if="isLoggedIn">
+        <v-btn color="primary"
+               x-large
+               class="my-4" block
+               @click="acceptInvitation">
+          {{ this.$tc('components.invitation.acceptCurrentAuth') }}
+        </v-btn>
+        <v-btn color="primary"
+               x-large
+               class="my-4" block
+               @click="useAnotherAccount">
+          {{ this.$tc('components.invitation.useOtherAuth') }}
+        </v-btn>
+      </div>
+      <div v-else>
+        <v-btn color="primary"
+               x-large
+               class="my-4" block
+               :to="loginLink">
+          {{ this.$tc('components.invitation.login') }}
+        </v-btn>
+        <v-btn color="primary"
+               x-large
+               class="my-4" block
+               :to="{ name: 'register' }">
+          {{ this.$tc('components.invitation.register') }}
+        </v-btn>
+      </div>
+      <v-btn color="red"
+             x-large
+             class="my-4" block>
+        {{ this.$tc('components.invitation.reject') }}
+      </v-btn>
+    </div>
+    <v-alert v-else type="error">
+      {{ this.$tc('components.invitation.notFound') }}
+    </v-alert>
+  </auth-container>
+</template>
+
+<script>
+import AuthContainer from '@/components/layout/AuthContainer'
+import { campRoute, loginRoute } from '@/router'
+
+function loginLink () {
+  return loginRoute(this.$route.fullPath)
+}
+
+export default {
+  name: 'Invitation',
+  components: { AuthContainer },
+  props: {
+    getCamp: { type: Function, required: true },
+    getCampCollaboration: { type: Function, required: true }
+  },
+  data () {
+    return {
+      camp: undefined,
+      campCollaboration: undefined
+    }
+  },
+  computed: {
+    campCollaborationFound () {
+      return this.campCollaboration !== null
+    },
+    campCollaborationOpen () {
+      if (!this.campCollaboration) {
+        return undefined
+      }
+      return this.campCollaboration.status === 'invited'
+    },
+    isLoggedIn () {
+      return this.$auth.isLoggedIn()
+    },
+    loginLink
+  },
+  mounted () {
+    const me = this
+    this.getCampCollaboration().then(function (campCollaboration) {
+      me.campCollaboration = campCollaboration
+    }, function () {
+      me.campCollaboration = null
+    })
+    this.getCamp().then(function (camp) {
+      me.camp = camp
+    }, function () {
+      me.campCollaboration = null
+    })
```

Comment:
Wenn du die Arrow Function Syntax verwendest fällt das spezielle Handling von `this` / `me` weg. Arrow functions verhalten sich in Bezug auf `this` anders als die klassischen anonymen Funktionen.
```suggestion
    this.getCampCollaboration().then(campCollaboration => {
      me.campCollaboration = campCollaboration
    }, () => {
      me.campCollaboration = null
    })
    this.getCamp().then(camp => {
      me.camp = camp
    }, () => {
      me.campCollaboration = null
    })
```

Reply:
habe es so gelöst, dass die campCollaborations jetzt als vuex-store proxy in die komponente hineinkommen,
und vo da die einzelne campCollaboration und das camp berechnet wird.